packages/lit-dev-content/samples/tutorials/intro-to-lit/06/after/todo-list.ts

```diff
@@ -15,13 +15,11 @@ export class ToDoList extends LitElement {
     }
   `;
 
-  @property()
+  @property({attribute: false})
```

Comment:
Great catch. This made me realize there are a couple other inconsistencies between Step 6 and Step 7.

In Step 6 there's indentation that changes on the solve.
Step 6 uses `class` whilst Step 7 uses `export class`.

However they aren't blocking. Maybe we track in an issue and resolve separately.

Reply:
Mea culpa here. It's really hard to maintain all the before/after versions (@e111077's recent changes should help a ton with that). Currently the `{attribute: false}` is also missing in the previous step where this property is added.

https://github.com/lit/lit.dev/blob/main/packages/lit-dev-content/samples/tutorials/intro-to-lit/05/before/todo-list.ts#L6

Looks like the everything _before_ step 7 omits the export statement.